libslide/src/emit.rs

```diff
@@ -41,6 +41,9 @@ bitflags::bitflags! {
         /// Emit divisions as fractions.
         /// Applies to LaTeX emit.
         const FRAC = 1;
+        /// Emit assignment operators as ":=".
+        /// Applies to pretty emit.
+        const DEFINE_ASSIGN = 2;
     }
```

Comment:
NIT: change FRAC to define_frac for consistency with these vars

Reply:
well := is the "definition assignment" operator

Reply:
whats the diff between assign and definition assignment. I don't think is a universal notation for this op. Like we see it in so many ways.